File: native/android/src/core/Theme.kt
package app.dovo.core

import androidx.compose.ui.graphics.Color

// "Mesa Nocturna" — valores EXACTOS de app/globals.css (sincronizar si cambian).
// Chrome calmado (papel/ink); color/motion solo en lo que está en juego.
object Dovo {
    val papel = Color(0xFFF4F4F6)       // near-white frío
    val ink = Color(0xFF08070D)         // blue-black
    val inkSoft = Color(0xFF2C2B36)
    val signal = Color(0xFF6D4AFF)      // violeta eléctrico — el único acento
    val signalDeep = Color(0xFF5A37E0)
    val signalOnGame = Color(0xFF8F70FF)
    val night1 = Color(0xFF16132A)      // superficies de juego (oscuras)
    val night2 = Color(0xFF0B0A14)
    val night3 = Color(0xFF07060D)
    val coop = Color(0xFF3AC49A)        // pacto/boost/escudo
    val rival = Color(0xFFFF3D5E)       // duelo/ataque/deuda
    val gold = Color(0xFFF0C75A)        // ceremonia (solo sobre oscuro)
}

File: native/ios/Sources/Core/DesignSystem/Theme.swift
import SwiftUI

// "Mesa Nocturna" — valores EXACTOS de app/globals.css (mantener sincronizados
// si cambian en la web). Tesis: chrome calmado (papel/ink); el color y el
// movimiento viven SOLO en lo que está en juego (trato/apuesta/resultado).
extension Color {
    static let dvPapel        = Color(dvHex: 0xF4F4F6) // near-white frío
    static let dvInk          = Color(dvHex: 0x08070D) // blue-black
    static let dvInkSoft      = Color(dvHex: 0x2C2B36)
    static let dvSignal       = Color(dvHex: 0x6D4AFF) // violeta eléctrico — el único acento
    static let dvSignalDeep   = Color(dvHex: 0x5A37E0)
    static let dvSignalOnGame = Color(dvHex: 0x8F70FF) // signal legible sobre panel oscuro
    static let dvNight1       = Color(dvHex: 0x16132A) // superficies de juego (oscuras)
    static let dvNight2       = Color(dvHex: 0x0B0A14)
    static let dvNight3       = Color(dvHex: 0x07060D)
    static let dvCoop         = Color(dvHex: 0x3AC49A) // pacto/boost/escudo
    static let dvRival        = Color(dvHex: 0xFF3D5E) // duelo/ataque/deuda
    static let dvGold         = Color(dvHex: 0xF0C75A) // ceremonia (solo sobre oscuro)
}

extension Color {
    init(dvHex hex: UInt32) {
        self.init(
            .sRGB,
            red: Double((hex >> 16) & 0xFF) / 255,
            green: Double((hex >> 8) & 0xFF) / 255,
            blue: Double(hex & 0xFF) / 255,
            opacity: 1
        )
    }
}
